src/iohandlers.cpp

```diff
@@ -36,9 +36,15 @@ static void print_addresses(const fds_data *data, int &list_count)
         getnameinfo(reinterpret_cast<const sockaddr *>(&data->server_addr), data->server_addr_len,
                 hbuf, sizeof(hbuf), pbuf, sizeof(pbuf),
                 NI_NUMERICHOST | NI_NUMERICSERV);
-        printf("[%2d] IP = %-15s PORT = %5s # %s\n", list_count++,
+        switch (data->server_addr.ss_family) {
+            case AF_UNIX:
+                printf("[%2d] Address is %5s # UDS\n", list_count++, pbuf);
+                break;
+            default:
```

Comment:
please consider using `PRINT_PROTOCOL()`
please check if you need `hbuf` instead of `pbuf` (see comment above)

Reply:
Added PRINT_SOCKET_TYPE for clarifying that this is not protocol (TCP/UDP) but rather a socket type